packages/react/src/components/MultiSelect/__tests__/MultiSelect-test.js

```diff
@@ -275,7 +275,7 @@ describe('MultiSelect', () => {
         />
       );
 
-      expect(document.getElementById('custom-id')).toBeTruthy();
+      expect(document.getElementById('custom-id-menu')).toBeTruthy();
```

Comment:
Does this mean that there is no element with the id `custom-id` available? I think this is the breaking change potentially